plugins/grpc/swift/v1.16.0/buf.plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/grpc/swift
+plugin_version: v1.16.0
+source_url: https://github.com/grpc/grpc-swift
+description: Generates Swift client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/apple/swift:v1.21.0
+output_languages:
+  - swift
+registry:
+  swift:
+    deps:
+      - source: https://github.com/grpc/grpc-swift.git
+        package: grpc-swift
+        products: [ GRPC ]
+        version: 1.16.0
```

Comment:
@pgmitche / @stefanvanburen - Does this look right? Just confirming since this is the first auto update of this plugin since we added the registry metadata.

Reply:
It looks like support was dropped for Swift 5.5: https://github.com/grpc/grpc-swift/releases/tag/1.16.0